openff/toolkit/utils/utils.py

```diff
@@ -209,7 +209,7 @@ def string_to_quantity(quantity_string) -> Union[str, int, float, unit.Quantity]
 
 def convert_all_strings_to_quantity(
     smirnoff_data: Dict,
-    ignore_keys: List[str] = list(),
+    ignore_keys: Iterable[str] = tuple(),
```

Comment:
This is a simpler fix than letting it be `Optional[List[str]]` and adding a layer of logic around it possibly being `None` vs. `list()`.